includes/internal/emails/class-email-list-table.php

```diff
@@ -230,4 +240,48 @@ private function get_row_actions( $post ): array {
 		 */
 		return apply_filters( 'sensei_email_list_row_actions', $actions, $post, $this );
 	}
+
+	/**
+	 * Display table content wrapped inside a form
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return void
+	 */
+	public function display() {
+		echo '<form id="posts-filter" action="' . esc_url( admin_url( 'edit.php' ) ) . '" method="get">';
+		parent::display();
+		echo '<input type="hidden" name="post_type" value="' . esc_attr( Email_Post_Type::POST_TYPE ) . '">';
+		wp_nonce_field( 'sensei_email_bulk_action' );
+		echo '</form>';
+	}
+
+	/**
+	 * Display the bulk actions.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @param string $which The location of the bulk actions: 'top' or 'bottom'.
+	 *
+	 * @return void
+	 */
+	public function bulk_actions( $which = '' ) {
+		if ( 'top' !== $which ) {
+			return;
```

Comment:
If we are following the WordPress convention, we should be showing the bulk actions at the bottom as well. This would be convenient when the list is too long. What do you think?

Reply:
Probably no harm in that, added here https://github.com/Automattic/sensei/pull/6552/commits/b0cf3ede2ff1055210d74829a71862c0222aa7fd